.github/workflows/test.yml

```diff
@@ -132,19 +123,7 @@ jobs:
         name: 🐿 Setup Golang
         uses: actions/setup-go@v4
         with:
-          go-version: "^1.20"
-      -
-        name: Get data from build cache
-        uses: actions/cache@v3
-        with:
-          path: |
-            ~/go/pkg/mod
-            ~/.cache/go-build
-            ~/Library/Caches/go-build
-            ~\AppData\Local\go-build
-          key: ${{ runner.os }}-go-docker-${{ hashFiles('**/go.sum') }}
-          restore-keys: |
-            ${{ runner.os }}-go-docker-
```

Comment:
not 100% sure this is going to preserve the docker caches, but down to try

Reply:
For docker we have caching in the action:

```
     -
        name: Build e2e image
        uses: docker/build-push-action@v4
        with:
          load: true
          context: .
          tags: osmosis:debug
          # Use experimental Cache backend API: 
          # https://github.com/docker/build-push-action/blob/master/docs/advanced/cache.md#cache-backend-api
          cache-from: type=gha
          cache-to: type=gha,mode=max
```